scripts/mkcloud

```diff
@@ -199,6 +201,21 @@ function error_exit()
     exit $exitcode
 } >&2
 
+function determine_host_mtu()
+{
+    local host_mtu
+
+    host_mtu=$(LC_ALL=C sort -n /sys/class/net/*/mtu | head -n 1)
+    # We may also have want_mtu_host so in that case we need to adjust
+    # it accordingly
+    if [[ $want_mtu_size -gt $host_mtu ]]; then
+        echo "Lowering vNIC MTU from $want_mtu_size to $host_mtu..."
+        want_mtu_size=$host_mtu
```

Comment:
most of the traffic stays between the VMs, which all use the high MTU value, so get enhanced performance from it
and traffic that goes to the outside world is covered by the set-mss IP-tables rule (though that might even not be needed, as there should be path-mtu-discovery)